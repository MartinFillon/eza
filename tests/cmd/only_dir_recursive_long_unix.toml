bin.name = "eza"
args = "tests/itest --recurse --long --no-user --no-time --no-filesize --no-permissions --only-dirs"
