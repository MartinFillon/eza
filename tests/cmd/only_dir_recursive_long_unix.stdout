exa
vagrant

tests/itest/exa:
sssssssssssssssssssssssssggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggsssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss

tests/itest/exa/sssssssssssssssssssssssssggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggsssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss:

tests/itest/vagrant:
debug
dev
log

tests/itest/vagrant/debug:

tests/itest/vagrant/dev:

tests/itest/vagrant/log:
run

tests/itest/vagrant/log/run:
